src/fs-fixture.ts

```diff
@@ -50,6 +50,16 @@ export class FsFixture {
 		});
 	}
 
+	/**
+	Copy a file to the fixture directory. Pass in a subpath destination.
+	*/
+	copyFile(filePath: string, subpath = '') {
```

Comment:
I don't think we can default the subpath to `''` since `fs.copyPath` expects a filename rather than a dir:
https://nodejs.org/api/fs.html#fspromisescopyfilesrc-dest-mode

Reply:
> I don't think we can default the subpath to `''` since `fs.copyPath` expects a filename rather than a dir: https://nodejs.org/api/fs.html#fspromisescopyfilesrc-dest-mode

Ah, you are correct. I assumed it, like `cp`, would copy into the given dir… Fixed now @privatenumber.